quiz.js

```diff
@@ -0,0 +1,58 @@
+
+var tree = {
```

Comment:
while this is an object we want to pass it into the grow tree function.  so it makes more sense to use the actual values in here.  Find me during class tonight and I can explain better.